TUTORIAL.md

```diff
@@ -507,18 +531,18 @@ main :: IO ()
 main = do
     manager <- newManager tlsManagerSettings
 
-    request <- parseUrl "http://httpbin.org/get"
+    request <- parseRequest_ "http://httpbin.org/get"
```

Comment:
I think you need either `let request = parseRequest_` or `request <- parseRequest`.